server/graphql/core/src/generic_filters.rs

```diff
@@ -9,6 +9,8 @@ pub struct StringFilterInput {
     equal_to: Option<String>,
     /// Search term must be included in search candidate (case insensitive)
     like: Option<String>,
+    /// Search term must begin with (case insensitive)
+    starts_with: Option<String>,
```

Comment:
have added starts_with to string filter.

Is this okay? I was wondering why we didn't have all implementations of StringFilter included already?